### PR TITLE
FOGL-3500 Fixed aiocoap import error when sending data to http protocol

### PR DESCRIPTION
**Pull request by rajeshkumar426**
Signed-off-by: rajeshkumar426 <rajeshkumar426@outlook.com>

### CODE REVIEW

**Review by praveen-garg — DISMISSED** [review]

File: extras/python/fogbench/__main__.py [hunk 1]
```diff
@@ -213,6 +210,9 @@ def read_out_file(_file=None, _keep=False, _iterations=1, _interval=0, send_to='
      URI "/other/sensor-values".
 
     """
+    from aiocoap import Context, Message, numbers
```
Comment (praveen-garg) [inline extras/python/fogbench/__main__.py:213]: @rajeshkumar426 do we need the `numbers` import?

File: extras/python/fogbench/__main__.py [hunk 2]
```diff
@@ -213,6 +210,9 @@ def read_out_file(_file=None, _keep=False, _iterations=1, _interval=0, send_to='
...
+    from cbor2 import dumps
+
     context = await Context.create_client_context()
 
     request = Message(payload=dumps(payload), code=POST)
```
Comment (praveen-garg) [inline extras/python/fogbench/__main__.py]: the POST will work withOUT any symbol import?

File: extras/python/fogbench/__main__.py (hunk 1 above)
Comment (rajeshkumar426) [reply extras/python/fogbench/__main__.py:213]: We need `numbers` import for `POST`.

File: extras/python/fogbench/__main__.py (hunk 2 above)
Comment (rajeshkumar426) [reply extras/python/fogbench/__main__.py]: No, `numbers` import required for `POST`. 
Fixed `numbers` import.

**Pull request closed without merging**